src/app.jsx

```diff
@@ -21,14 +20,12 @@ class App extends React.Component {
   async getProduct(id = '19093') {
     const [firstRes, secondRes, thirdRes] = await Promise.all([
```

Comment:
looks like the destructuring assignment is taking 3 params, but you're only dealing with 2 below